pkg/ebpf/events_definitions.go

```diff
@@ -6248,4 +6249,18 @@ var EventsDefinitions = map[int32]EventDefinition{
 			{Type: "u64", Name: "post_handler_addr"},
 		},
 	},
+	CallUsermodeHelperEventID: {
+		ID32Bit: sys32undefined,
+		Name:    "call_usermodehelper",
+		Probes: []probe{
+			{event: "call_usermodehelper", attach: kprobe, fn: "trace_call_usermodehelper"},
+		},
+		Sets: []string{},
+		Params: []trace.ArgMeta{
+			{Type: "const char*", Name: "pathname"},
+			{Type: "const char*const*", Name: "argv"},
+			{Type: "const char*const*", Name: "envp"},
```

Comment:
Differently than execve probe, in here we always want to get the env variables I suppose. Mostly because it is not a question of tracing executed commands, but a specific security probe. Spoke to @itamarmaouda101 who confirmed it.